src/node/recursiveModuleCompile.js

```diff
@@ -50,6 +50,8 @@ function recursiveModuleCompileToSingleFile(outputFile, includes, options) {
     return include;
   });
 
+  options.rootModules = resolvedIncludes;
```

Comment:
I wonder if we could pass this information in some other way? It feels a bit odd to put this on the options object.